source/tests/yats-test/taskcontainer_test.cpp

```diff
@@ -0,0 +1,23 @@
+#include <gmock/gmock.h>
+
+#include <yats/slot.h>
+#include <yats/task_container.h>
+#include <yats/scheduler.h>
+
+#include <memory>
+
+TEST(taskcontainer_test, return_parameters)
```

Comment:
Should be task_container_test, to be consistent with task_container

Reply:
taskconfigurator does it this way too.